controller/linodemachine_controller.go

```diff
@@ -308,6 +310,10 @@ func (r *LinodeMachineReconciler) reconcileCreate(
 
 		linodeInstance, err = machineScope.LinodeClient.CreateInstance(ctx, *createOpts)
 		if err != nil {
+			if linodego.ErrHasStatus(err, linodeTooManyRequests) || linodego.ErrHasStatus(err, linodeEOFOrTimeout) {
+				logger.Error(err, "Failed to create Linode instance due to API error, requeing")
+				return ctrl.Result{RequeueAfter: reconciler.DefaultMachineControllerWaitForRunningDelay}, nil
```

Comment:
Question: should I use a different delay here? Something like `DefaultMachineControllerRetryDelay`?

Reply:
Yeah, my vote would be for `DefaultMachineControllerRetryDelay` since this is different from waiting for the linode to be running. For something like a 429 this makes me think we might want an exponential backoff but just a separate variable is fine for now.

Reply:
updated.